scripts/3scale-mock

```diff
@@ -144,7 +144,7 @@ if __name__ == "__main__":
             args = []
         curl_command = "curl"
         curl_command += ' -H "x-rh-identity: %s"' % encoded_identity
-        curl_command += " %s" % " ".join("%s" % s if not True in [c.isspace() or c in [";", "&"] for c in s] else "'%s'" % s for s in args)
+        curl_command += " %s" % " ".join("%s" % s if False in [c.isspace() or c in [";", "&"] for c in s] else "'%s'" % s for s in args)
```

Comment:
I have no idea what this does 😄 depending on the intention, shouldn't it be `if True not in [...`?